trestle/core/profile_resolver.py

```diff
@@ -169,12 +238,17 @@ def _prune_catalog(self) -> cat.Catalog:
             # if a control includes controls - only include those that we know are needed
             final_control_ids = self._prune_controls(needed_ids)
 
+            cat_controls = []
+
             # build the needed groups of controls
             group_dict: Dict[str, cat.Group] = {}
             for control_id in final_control_ids:
+                control = self._catalog_interface.get_control(control_id)
                 group_id, group_title, group_class = self._catalog_interface.get_group_info(control_id)
+                if group_id == 'catalog':
```

Comment:
Should be CATALOG ?

Reply:
This is an internal way to keep track of controls that are not in a group - they are in the catalog's own list of controls.  So it is correct but I will make a note to comment it.